.github/workflows/buildAndTest.yml

```diff
@@ -27,6 +27,27 @@ jobs:
           fetch-depth: 2
           submodules: "true"
 
+      - name: Install necessary build tools for libelf
```

Comment:
We recently decided to replace our custom libelf with the pre-built one. It means these tools are no longer necessary. I don't know if it's better to make that change in a different commit or to edit this one. What do you think?

Reply:
I can edit the commit. Thanks for the comments I see how you are doing it in your mlir-aie fork.

Reply:
In the moment felt better as a new commit so put in [04a27d4](https://github.com/Xilinx/mlir-air/pull/367/commits/04a27d44a94310b94a27d0ff068b683d38f215f9). Let me know what you think. I couldn't get CMake to automatically find LibElf if you know how to do that that could make it cleaner, but I am just pointing to the standard install location which also works.

Reply:
I don't know how to get CMake to search for a library. We could pass the path explicitly (like I was doing for the custom version) but with a default to the standard Ubuntu location. I think for now, relying on the standard Ubuntu location (as you are doing) is acceptable. If everyone will just use 'apt install' then it is highly unlikely the library will be at a different location.

Reply:
100% agree. Sweet will merge this in. Thanks!